### PR TITLE
deps(go-piper): update to 2023.11.6-3

### PR DESCRIPTION
**Pull request by M0Rf30**
**Description**

This PR fixes go-piper compilation

**Notes for Reviewers**
It is linked to https://github.com/mudler/go-piper/pull/1
Tested on a complete rebuild with GO_TAGS="tts" and the example reported in the official docs
All the *.so
and required backend assets (espeak-ng-data) will live under go-piper/piper/build/pi 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -142,7 +144,7 @@ gpt4all:
 
 ## go-piper
 go-piper:
-	git clone --recurse-submodules https://github.com/mudler/go-piper go-piper
+	git clone --recurse-submodules https://github.com/M0Rf30/go-piper go-piper
```
Comment (mudler) [inline Makefile:147]:
```suggestion
	git clone --recurse-submodules https://github.com/mudler/go-piper go-piper
```

**Comment by mudler** [comment]
@M0Rf30 somehow the pinning of  the go-piper commit was overwritten

**Comment by M0Rf30** [comment]
> this is looking good! however @M0Rf30 somehow the pinning of the go-piper commit was overwritten, without it will still use the old go-piper version

I removed any reference to updated go-piper commit. I've seen that the bot automatically update the commits, looking at the other repos. 
Please give me suggestions on how to manage this. Should I bump the commit?should  wait for go-piper PR to be merged?

**Comment by M0Rf30** [comment]
Here we go. I updated also the commit

**Review by mudler — DISMISSED** [review]

**Comment by M0Rf30** [comment]
Ok got it, I need to rework the CI test steps. otherwise stable-diffusion will always fail

**Comment by M0Rf30** [comment]
CGO for flags for piper now have a specific name in the Makefile. this avoid conflicts on unneeded libs/includes for stablediffusion when both GO_TAGS are called.

ESPEAK_DATA now points to piper shipped resources and go-piper is a mandatory dep for backend-assets/espeak-ng-data

**Review by mudler — APPROVED** [review]
Nice contribution, thanks @M0Rf30 for keeping trying :)

**Pull request merged**